scripts/data_overlap/compute_data_overlap_metrics.py

```diff
@@ -110,7 +110,7 @@ def create_ngram_index(
 
                 # concatenate the last n-1 tokens of input and the first n-1 tokens of reference and compute n-grams on this "interesection token" sequence
                 # for instance: input = ["is 2+2 4 true or false"] reference = ["true"]
-                # the intersection is the 5-gram ["4 true or false true"] 
+                # the intersection is the 5-gram ["4 true or false true"]
```

Comment:
I think this is good (as we discussed), but (i) do we have to get providers to rerun with this new code (non-trivial cost) and (ii) I wonder how often the question and answer will be juxtaposed.  If there's any token that separates the Q and A, then we won't detect overlap.

Reply:
Yeah I think for now we can just run this for the pile and see what sort of insights we get; this is simply an additional metric that doesn't affect existing metrics. If we're concerned for a token gap, we can allow for a skip token budget, esp for this metric, though it may be premature to add at this point